modules/scan-inactive-images.adoc

```diff
@@ -2,15 +2,27 @@
 //
 // * operating/manage-vulnerabilities.adoc
 // * operating/examine-images-for-vulnerabilities.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="scan-inactive-images_{context}"]
 = Scanning inactive images
 
-{product-title} scans all active (deployed) images every 4 hours and updates the image scan results to reflect the latest vulnerability definitions.
+{rh-rhacs-first} scans all active (deployed) images every 4 hours and updates the image scan results to reflect the latest vulnerability definitions.
 //TODO: add link to Re-scanning images.
 
-You can also configure {product-title} to scan inactive (not deployed) images automatically.
+You can also configure {product-title-short} to scan inactive (not deployed) images automatically.
 
+
+.Procedure
+
+. On the RHACS portal, navigate to *Vulnerability Management > Dashboard*.
+. On the *Dashboard* view header, select *IMAGES*.
+. Click *MANAGE WATCHES* to manage the scanning of watched images.
+. In the *MANAGE WATCHED IMAGES* dialog, enter the inactive image’s name, and not the image `id`, for which you want to enable scanning. Image name is the fully-qualified image name, beginning with the registry and ending with the tag.
```

Comment:
```suggestion
. In the *MANAGE WATCHED IMAGES* dialog, enter the inactive image’s name (not the image `id`), for which you want to enable scanning. Image name is the fully-qualified image name, beginning with the registry and ending with the tag.
```

[nit]
I'm not sure if this fits the Red Hat style, but I was confused about the wording (it makes it sound like "enter name AND NOT id" which wouldn't make sense)

Reply:
**nitpick:** I agree, maybe:
```
In the *MANAGE WATCHED IMAGES* dialog, enter the inactive image’s name for which you want to enable scanning. Verify that you enter the name of the image and not the image `id`. Image name is the fully-qualified image name, beginning with the registry and ending with the tag.
```